dockerfile-image-update/pom.xml

```diff
@@ -122,6 +122,11 @@
             <artifactId>github-api</artifactId>
             <version>1.303</version>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-s3</artifactId>
+            <version>1.11.641</version>
```

Comment:
See https://github.com/aws/aws-sdk-java-v2/#using-the-sdk .  We should probably use the latest version of the SDK unless we have good reason not to.